pupil-spa/src/app/question/question.component.spec.ts

```diff
@@ -18,7 +25,9 @@ describe('QuestionComponent', () => {
     component = fixture.componentInstance;
     fixture.detectChanges();
     // prevent Timeout's being set
-    spyOn(component, 'ngAfterViewInit').and.returnValue(null);
+    // NOTE - calling this makes no difference to sendTimeoutEvent test.
+    // all other tests work without the call below.
+    //spyOn(component, 'ngAfterViewInit').and.returnValue(null);
```

Comment:
If you run the tests in a browser, e.g. Chrome, with the console open, I suspect that you will see timeout's happening in the background (if you configure suitable logging).  It was only in there to shut these down.